Dockerfile

```diff
@@ -24,19 +24,23 @@ RUN apt-get update && \
     ca-certificates
 
 # Retrieve Grid2Op
-RUN git clone https://github.com/rte-france/Grid2Op
+RUN git clone https://github.com/bdonnot/Grid2Op
```

Comment:
Restore to `https://github.com/rte-france/Grid2Op` for the main docker container